recodex-api.spec

```diff
@@ -0,0 +1,100 @@
+%define name recodex-api
+%define short_name api
+%define version 1.20.0
+%define unmangled_version d59975357c86fd09f4222aa28f0d5bcdfca05d04
+%define release 1
+
+Summary: ReCodEx API component
+Name: %{name}
+Version: %{version}
+Release: %{release}
+License: MIT
+Group: Development/Libraries
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-buildroot
+Prefix: %{_prefix}
+Vendor: Petr Stefan <UNKNOWN>
+Url: https://github.com/ReCodEx/api
+Requires(post): systemd
+Requires(preun): systemd
+Requires(postun): systemd
+Requires: git php73-php php73-php-json php73-php-mysqlnd php73-php-ldap php73-php-pecl-zip php73-php-pecl-zmq php73-php-xml php73-php-intl php73-php-mbstring
```

Comment:
First of all, I am concerned that these PHP packages are present only in remi repo. Is there a way how to specify repo-dependencies?
Furthermore, I think we are missing `php73-php-pecl-yaml` package in this list.